demo/stories/10-announcement.stories.mdx

```diff
@@ -0,0 +1,234 @@
+
+Web Components standards have finally matured, and today, there are many ways to build and consume your Web Components, like: [Angular Elements](https://angular.io/guide/elements), [Stencil](https://stenciljs.com/docs/introduction/), [Vue](https://vuejsdevelopers.com/2018/05/21/vue-js-web-component/), [Svelte](https://svelte.dev/) and many, many more. And with the release of the new [Chromium Edge](https://www.theverge.com/2020/1/15/21066767/microsoft-edge-chromium-new-browser-windows-mac-download-os), this means that all major browsers support Web Components natively.
+
+So today, I'm happy to share with you all that ING is open-sourcing its own core library of Web Components: Lion! 🎉
+
+## Why Lion?
+
+Imagine the following fictional scenario:
+
+> Alfred works at Betatech, and is working on a new in-house application.
+
+### Agnostic
+
+Finding components is hard, and can lead to a lot of frustration.
+
+> Alfred finds a perfect component for his use case, but unfortunately, it's written in a specific JavaScript framework, and Betatech uses a different framework, so sadly he's not able to use it.
+
+Additionally, companies have their own visual identities and overriding fully styled-components to make them look like the companies identity, can sometimes be more work than start styling from scratch.
+
+> Alfred finds a component that offers all the needed functionalities, however, it looks vastly different from Betatechs identity.
+
+### Customize Functionality
+
+Once you've found your perfect component, at some point down the line you may discover that the component lacks some kind of functionality that you would like to achieve.
+
+> Alfred decides to copy over the code of a component he found, and implement the feature himself, and as a result now has to maintain the entire component as well, which becomes an added burden to the project.
+
+### Accessibility
+
+Your components should be usable by every kind of user. Accessibility is hard to get right, but an essential feature, additionally, accessibility issues can be impossible to fix at a later stage without breaking changes, so it is crucial to incorporate accessibility right from the start.
+
+> Somewhere down the line, Alfred finds an accessibility issue with a component he found online. He can't fix this issue in his code, so he asks the maintainers to help him out. They tell him they can't fix it because they don't want to do a breaking change.
+
+Or, alternatively
+
+> Alfred writes his own component, and the accessibility issue is hard to fix because of the way he initially wrote his HTML structure, and will also lead to a breaking change.
+
+### Summary
+
+Does Alfred's story feel relatable?
+
+It highlights multiple common issues in modern web development:
+- Finding and adding a dependency is a big deal
```

Comment:
Would it be clear for the reader why? 
 - Finding is hard because they are never white label or high quality?
 - Adding is a big deal because of filesize/network perf?